config/dpkg/control

```diff
@@ -1,14 +1,23 @@
-Source: python-artifacts
+Source: artifacts
 Section: python
 Priority: extra
 Maintainer: Forensic artifacts <forensicartifacts@googlegroups.com>
-Build-Depends: debhelper (>= 7.0.0), python, python-setuptools
-Standards-Version: 3.9.2
+Build-Depends: debhelper (>= 7), python-all (>= 2.7~), python-setuptools, python3-all (>= 3.2~), python3-setuptools
```

Comment:
The idea is to use it with plaso, no need to re-write the registry and reading functionality. 
https://github.com/joachimmetz/plaso/blob/artifacts/plaso/engine/engine.py#L94